ur_description/urdf/ur.ros2_control.xacro

```diff
@@ -29,7 +29,7 @@
           <param name="servoj_gain">2000</param>
           <param name="servoj_lookahead_time">0.03</param>
           <param name="use_tool_communication">0</param>
-          <param name="kinematics/hash">"${hash_kinematics}"</param>
+          <param name="kinematics/hash">${hash_kinematics}</param>
```

Comment:
This was causing the trouble mentioned in https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/302#issuecomment-1049026179.